tests/pipelines/unclip/test_unclip.py

```diff
@@ -363,6 +362,41 @@ class DummyScheduler:
         assert np.abs(image - image_from_text).max() < 1e-4
 
 
+# @nightly
+class UnCLIPPipelineCPUIntegrationTests(unittest.TestCase):
+    def tearDown(self):
+        # clean up the VRAM after each test
+        super().tearDown()
+        gc.collect()
+        torch.cuda.empty_cache()
+
+    def test_unclip_karlo_cpu_fp32(self):
+        expected_image = load_numpy(
+            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main"
+            "/unclip/karlo_v1_alpha_horse_cpu.npy"
+        )
+
+        pipeline = UnCLIPPipeline.from_pretrained("kakaobrain/karlo-v1-alpha")
+        pipeline.set_progress_bar_config(disable=None)
+
+        generator = torch.manual_seed(0)
+        output = pipeline(
+            "horse",
+            num_images_per_prompt=1,
+            generator=generator,
+            output_type="np",
+        )
+
+        image = output.images[0]
+
+        np.save("/home/patrick_huggingface_co/diffusers-images/unclip/karlo_v1_alpha_horse_cpu.npy", image)
+        images = pipeline.numpy_to_pil(image)
+        images[0].save("/home/patrick_huggingface_co/diffusers-images/unclip/karlo_v1_alpha_horse_image.png")
+
+        assert image.shape == (256, 256, 3)
+        assert np.abs(expected_image - image).max() < 1e-1
```

Comment:
Even on CPU I cannot get a higher precision

Reply:
@anton-l I'd say we make this a nightly ? It takes 5 min on CPU

Reply:
Yeah, it should be ok to move the tests for less frequently updated pipelines completely to nightly IMO